internal/gitcontext/branchfiles_test.go

```
@@ -0,0 +1,400 @@
+}
+
+func TestRunGitErrorBranches(t *testing.T) {
+	r := newRepo(t)
+	base := strings.TrimSpace(r.git("rev-parse", "HEAD"))
+	for i := 0; i < 6; i++ {
+		r.write(fmt.Sprintf("src/file%02d.txt", i), filler(fmt.Sprintf("s%02d", i), 25_000))
+	}
+	r.commit("big change")
+
+	for _, failOn := range []string{"--name-only", "--no-textconv"} {
+		failOn := failOn
```

golangci-lint (2.12.2)

[error] The copy of the 'for' variable "failOn" can be deleted (Go 1.22+)

(copyloopvar)